.github/workflows/code-style.yml

```diff
@@ -75,17 +75,26 @@ jobs:
                 with:
                     composer-options: --ansi --prefer-dist
 
-            -   name: Fix code style
+            -   name: Rector Rectify
                 if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name == github.repository
                 run: vendor/bin/rector process --no-progress-bar
 
-            -   name: Commit changed files
+            -   name: Commit rector changes
                 uses: stefanzweifel/git-auto-commit-action@v4
                 if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name == github.repository
                 with:
                     commit_message: Apply rector changes
                     commit_user_name: rex-bot
                     commit_user_email: rex-bot@users.noreply.github.com
 
-            -   name: Check code style
-                run: vendor/bin/rector process --ansi --no-progress-bar --dry-run
```

Comment:
Der doppelte Check hier hat schon seinen Zweck (wenn ich es richtig sehe, hast du ihn ersatzlos entfernt..).

Einerseits soll er prüfen, ob nach dem Commit auch wirklich keine weiteren Änderungen gemacht werden.
(Darauf könnte man vielleicht verzichten.)

Und andererseits, wichtiger, ist es in Fork-PRs der einzige Check, denn dort entfällt der obere Check&Fix-Step, da dort ja nicht gefixt und commitet werden kann.